Mergin/utils.py

```diff
@@ -903,11 +903,13 @@ def has_schema_change(mp, layer):
     tmp_file.close()
     geodiff.schema('sqlite', '', local_path, tmp_file.name)
     with open(tmp_file.name, encoding="utf-8") as f:
-        base_schema = json.load(f).get('geodiff_schema')
+        data = f.read()
+        base_schema = json.loads(data.replace("\n", "")).get('geodiff_schema')
```

Comment:
I would revert this workaround

Reply:
But without it it would be impossible to sync projects with such field names, as Python won't be able to parse JSON file and show an error message.